Jenkinsfile

```diff
@@ -139,10 +139,11 @@ pipeline {
         stage('Tests') {
             parallel {
                 stage('Unit') {
-                    agent any
+                    agent { label "gelman-group-mac" }
```

Comment:
@seantalts @wds15 how are we doing the unit tests for MPI? I think it would be a good idea to break this out so all the GPU tests run in a seperate stage